usaspending_api/settings.py

```diff
@@ -227,6 +227,63 @@
     },
 }
 
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'default-loc-mem-cache',
+    },
+    'locations': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'locations-loc-mem-cache',
+    },
+}
+
+# Cache environment - 'local', 'disabled', or 'elasticache'
+CACHE_ENVIRONMENT = 'local'
+
+# Set up the appropriate elasticache for our environment
+CACHE_ENVIRONMENTS = {
+    # Elasticache settings are changed during deployment, or can be set manually
+    'elasticache': {
+        'BACKEND': 'django_elasticache.memcached.ElastiCache',
+        'LOCATION': 'ELASTICACHE-CONNECTION-STRING',
+        'TIMEOUT': 'TIMEOUT-IN-SECONDS',
+        'OPTIONS': {
+            'IGNORE_CLUSTER_ERRORS': True,
+        },
+    },
+    'local': {
```

Comment:
Do `BACKEND` and `LOCATION` here replace the settings in `CACHES` (line 231). Or is this in addition to? If the former, should we delete `CACHES`? If the latter, what do you think about adding some additional comments here that explain how these work together? If I understand correctly, we're caching some stuff (e.g., API responses) in memcache and other things (data load info) in local memory. But I'm not sure how all of these knobs and dials work together.